README.md

```diff
@@ -23,8 +23,40 @@ refer to the [roadmap] and the issue tracker for more details.
 
 This repository provides pre-built binaries that are signed and published
 using [GoReleaser]. The signature for these binaries are generated using
-[Sigstore], using the release workflow's identity. Please use release v0.1.0 or
-higher, as prior releases were created to test the release workflow.
+[Sigstore], using the release workflow's identity. Make sure you have
+[cosign] installed on your system, then you will be able to securely download
+and verify the gittuf release:
+
+```sh
+# Modify these values as necessary.
+# One of: amd64, arm64
+ARCH=amd64
+# One of: linux, darwin, freebsd, windows
+OS=linux
+# https://github.com/gittuf/gittuf/releases
+VERSION=0.1.0
+cd $(mktemp -d)
+
+curl -LO https://github.com/gittuf/gittuf/releases/download/v${VERSION}/gittuf_${VERSION}_${OS}_${ARCH}
```

Comment:
Hm, I guess that's counter to using `sudo install` though anyway.

Reply:
Yeah I briefly thought about that as well. It might provide some value for `cygwin` users.
Maybe drop `windows` from the suggested `OS` values? It probably required a different script / approach altogether.